lib/ReactViews/DataCatalog/CatalogItem.jsx

```diff
@@ -10,13 +10,15 @@ import Styles from './data-catalog-item.scss';
 const STATE_TO_TITLE = {
     loading: 'Loading...',
     remove: 'Remove',
-    add: 'Add'
+    add: 'Add',
+    trash: 'delete dataset'
```

Comment:
Casing should be consistent with other titles. Also it might be a group rather than a dataset. And I think Remove makes it more clear that you're removing it from this view rather than deleting it completely from your computer or whatever.

```suggestion
    trash: 'Remove'
```